### DISCUSSION TITLE
Broken merge confidence badges

### DISCUSSION THREAD

**Discussion by Mitgenosse** [renovatebot/renovate#31201] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab, Renovate 38.67.1

### Please tell us more about your question or problem

Hello everyone!

When renovate is creating a merge request to update a npm dependency, the merge confidence badges are not showing correctly anymore. This worked before.

In the example it's trying to update tslib from 2.6.3 to 2.7.0. The merge confidence badges do not show. The GitHub changelog is fetched and displayed correctly. It is creating links to the badges like the adoption badge as such: "https://badges.renovateapi.com/packages/npm/tslib/2.7.0/adoption-slim". This link shows a 504 as a Cloudflare issue. I receive the 504 error from my laptop and my personal phone, so I don't think it's a VPN issue from my side.

Did anything change from v37 to v38 that I missed?

![Screenshot 2024-09-04 at 12 19 21](https://github.com/user-attachments/assets/bf5a698c-aff0-43ab-809c-aa0108a857f6)

The error:

![Screenshot 2024-09-04 at 12 21 57](https://github.com/user-attachments/assets/df799166-ec26-4db8-a1d4-b9ce28819e7d)

The repo's renovate.json is extending this

```
"extends": [
    "config:base",
    ":maintainLockFilesWeekly",
    ":rebaseStalePrs",
    ":semanticCommits",
    ":semanticCommitScope(deps)",
    "docker:enableMajor",
    "docker:pinDigests",
    "group:linters",
    "mergeConfidence:all-badges"
  ],
```

Any ideas?

### Logs (if relevant)

I don't find any errors or anything suspicious in the logs.

**Comment by rarkins** [comment]
badges.renovateapi.com is a deprecated server. You should find the badges server to now be developer.mend.io. That old URL is no longer in the Renovate codebase so I can only guess that you have copy pasted it into a config of your own

**Comment by nabeelsaabna** [comment]
since `mergeConfidence:all-badges` is a builtin Renovate config option you should remove any custom definitions that contains `https://badges.renovateapi.com`

**Comment by Mitgenosse** [comment]
Thank you so much for informing me about this deprecation! I found a old merge-confidence-beta configuration which had the deprecated server configured. Now it's working as expected.

![Screenshot 2024-09-04 at 13 27 55](https://github.com/user-attachments/assets/b8d0a96a-a83d-45a3-b457-322bc785e2ca)

I'm a huge fan of the RenovateBot project, it's so awesome. Thank you so much!
